Source/Extensions/Blazorise.Splitter/SplitterSection.razor.cs

```diff
@@ -0,0 +1,68 @@
+﻿#region Using directives
+using System;
+using Blazorise.Utilities;
+using Microsoft.AspNetCore.Components;
+#endregion
+
+namespace Blazorise.Splitter;
+
+/// <summary>
+/// A resizable section of a <see cref="Splitter"/> component
+/// </summary>
+public partial class SplitterSection : BaseComponent, IDisposable
+{
+    #region Methods
+
+    /// <inheritdoc />
+    protected override void OnInitialized()
+    {
+        if ( Parent is null )
+            throw new ArgumentNullException( nameof( Parent ), "SplitSection must exist within a Split" );
+
+        base.OnInitialized();
+    }
+
+    /// <inheritdoc />
+    protected override void OnAfterRender( bool firstRender )
+    {
+        if ( firstRender )
+            Parent.RegisterSection( ElementRef );
+
+        base.OnAfterRender( firstRender );
+    }
+
+    /// <inheritdoc />
+    protected override void BuildClasses( ClassBuilder builder )
+    {
+        builder.Append( "split-section" );
```

Comment:
No need for prefixes.

Reply:
Generally, I try to stick to the naming of the library that we use behind the scenes. In this case, there is no prefix because the split library doesn't have it. Similarly is the case for Bootstrap, Bulma, antdesign. They all should have naming conventions for each of them separately.

The `b-` prefix should be used only in `Blazorise.csproj`, a core library. This is because in there we don't have any knowledge of which CSS framework is running. So we add a prefix to make it more explicit.

Reply:
Gotcha going to update.

> The b- prefix should be used only in Blazorise.csproj, a core library. This is because in there we don't have any knowledge of which CSS framework is running. So we add a prefix to make it more explicit.

Uhm, I'm not sure you explained this correctly, with the class builder abstraction you can make it so whatever css framework is running will apply correctly, in fact, that's precisely what you do independent of the library. But I understand what you mean.

Reply:
Also we've already used b- outside of the core library, including, for example cropper js which I believe uses a third party library:
![image](https://github.com/Megabit/Blazorise/assets/22283161/8e2bb27c-5a9a-4e1a-96cc-b0e09c3793ce)
So I'm not sure we are exactly following whatever convention you intended us to follow.
And even with the explanation you just gave, I'm still not 100% clear.

Reply:
Anyway, updated with this for now:
![image](https://github.com/Megabit/Blazorise/assets/22283161/bb1d25c5-d9b5-40d0-9482-7db9141bad26)

https://github.com/Megabit/Blazorise/wiki/Contributing

Reply:
@stsrki Just a reminder that the classes I was advocating to prefix **b-** I believe aren't from the 3rd party library, but "our own"
![image](https://github.com/Megabit/Blazorise/assets/22283161/7358c05a-fb91-4ca8-8109-4d83806ec259)

Reply:
True, but there is only one splitter component. If there were multiple then the prefix would make sense.

Reply:
Haha, I know it's confusing. I should have stuck to one naming convention in the past. We can discuss this in our next meeting for sure.

As I said, `b-` prefix should **generally** be used for stuff where we don't have other means to distinguish a CSS selector or need to add more context to the component. For example, DataGrid does this because it is shared between different frameworks. Autocomplete also. They both use the core Blazorise components.

Cropper is a standalone library. I chose prefixes on the fly basically. Maybe it is a good maybe not so good choice 💁‍♂️

Reply:
Yep still don't understand the convention. 
Either it's just me not understanding, or the convention isn't easy to follow.

As a guideline for people contributing to follow, I'm not sure they'll get it right.
But as long as you're reviewing PRs it's ok, I guess... haha

Reply:
That's what I'm telling the entire time, haha. There is only _general_ convention. But mostly, I go with the flow.

For really naming convention, we will both have to sit down and discuss it. It is beyond this PR.